README.md

```diff
@@ -0,0 +1,87 @@
+# Music Library 🎶
+
+**First project with [Manchester Codes](https://www.manchestercodes.com/) BACKEND module**
+
+The purpose of this project was to apply all learned aspects together in one project by building an API (Application Programming Interface) app to manage a database, a music library.
+
+The project followed TDD principles by writing a failing test and write the code to pass the test. This time we used Mocha supertest, an integration test which checks that app is communicating with the database.
+
+---
+
+**Consepts**
+
+- Database Design
+- SQL
+- MySQL
+- CRUD (CREATE, READ, UPDATE, DELETE) Operations
+
+---
+
+**Programming langages and technologies used**
+
+- Express, NodeJS
+- MySQL
+- Mocha /Chai supertest
+- Docker
+- Postman
+- MySQL workbench
+
+---
+
+**Set up**
+
+To be able to run this application follwing environment needs to be installed:
+
+[Docker](https://docs.docker.com/get-docker/)
+
+[MySQLWorkbench](https://www.mysql.com/products/workbench/)
+
+[Postman](https://www.postman.com/)
+
+```
+npm init -y
+```
+
+```
+npx eslint --init
+```
+
+```
+npm i -S express
+```
+
+```
+npm i -D nodemon
+```
+
+```
+npm i -D dotenv
+```
+
+```
+npm i -S mysql2
+```
+
+```
+npm i -D mocha chai supertest
+```
```

Comment:
Given that all these packages are already included in the `package.json` (and will be installed to the version specified in there when running `npm i`) these commands do not need to be run or included here. It would be worth including a line simply stating that running `npm i` is enough to get the project up and running from a code perspective. 
Also some notes on needing a database to be setup would be a good idea